src/containers/ConnectorManagement.jsx

```diff
@@ -83,13 +84,21 @@ export default class ConnectorManagement extends Component {
               <div className='installing-spinner' />
               <div>{t('working')}</div>
             </div>
-            : <AccountConnection
-              account={isConnected ? accounts[selectedAccount] : null}
-              onError={(error) => this.handleError(error)}
-              onSuccess={(account, messages) => this.handleSuccess(account, messages)}
-              onCancel={() => this.gotoParent()}
-              {...this.state}
-              {...this.context} />
+            : jobInBackground
+              ? <AccountSuccessTimeout
+                account={isConnected ? accounts[selectedAccount] : null}
```

Comment:
`isConnected` should always be `true` here, isn't it? This is a non-sense if we can have an `AccountSuccessTimeout` component while `isConnected` is false. I would use directly accounts[selectedAccount], what do you think @gregorylegarec ?